docs/modules/clients/pages/dotnet.adoc

```diff
@@ -2,22 +2,131 @@
+== Overview
+
+This section provides information about the .NET client for Hazelcast, and explains how to install and get started with the client. 
 
+TIP: To learn how to get started quickly with the Hazelcast .NET client, follow our simple xref:clients:csharp-client-getting-started.adoc[Get started with .NET] tutorial.  
 
-The Hazelcast native .NET client is an official library that allows .NET applications to connect to and interact with Hazelcast clusters. The key features and benefits include:
+The official Hazelcast .NET client allows .NET applications to connect to and interact with Hazelcast clusters. 
+The key features and benefits include:
 
-* Distributed Data Structures: it provides access to Hazelcast's distributed data structures such as maps, queues, topics, and more
-* SQL Support: ihe client allows running SQL queries over distributed data
-* Near Cache: it supports Near Cache for faster read speeds. Eventual consistency is also supported
-* Security Features: the client offers SSL support and Mutual Authentication for enterprise-level security needs
-* JSON Object Support: it allows using and querying JSON objects
-* Zero Downtime Upgrades: Blue/Green failover for zero downtime during upgrades is supported
-* Scalability: the Hazelcast .NET Client is designed to scale up to hundreds of members and thousands of clients, making it suitable for large-scale applications
+* Distributed Data Structures: it provides access to Hazelcast's distributed data structures such as maps, queues, topics, and more.
+* SQL Support: the client allows running SQL queries over distributed data.
+* Near Cache: it supports Near Cache for faster read speeds. Eventual consistency is also supported.
+* Security Features: the client offers SSL support and Mutual Authentication for enterprise-level security needs.
+* JSON Object Support: it allows using and querying JSON objects.
+* Zero Downtime Upgrades: Blue/Green failover for zero downtime during upgrades is supported.
+* Scalability: the Hazelcast .NET Client is designed to scale up to hundreds of members and thousands of clients, making it suitable for large-scale applications.
 
 These features make the Hazelcast .NET Client ideal for .NET applications requiring high-performance, scalable, and distributed data processing capabilities.
 
+TIP: For the latest .NET API documentation, see http://hazelcast.github.io/hazelcast-csharp-client/{page-latest-supported-csharp-client}/api/index.html[Hazelcast .NET Client docs].
+
+== Install the .NET client
+
+This section explains how to download and install the Hazelcast .NET client using the published Hazelcast.Net package from NuGet. 
+
+=== Requirements
+
+The Hazelcast .NET client is distributed as a NuGet package that targets .NET Standard versions 2.0 and 2.1, and specific .NET versions starting with 5.0. It can therefore be used in any application targetting .NET versions that support these .NET Standard versions:
```

Comment:
Our latest version of the client is supporting .NET 6 and .NET 8 besides .NET Framework 4.6.2 and 4.8. We drop supporting when Microsoft drops the .NET version. 5 was dropped a while ago.